loopout.go

```diff
@@ -77,10 +77,19 @@ func newLoopOutSwap(globalCtx context.Context, cfg *swapConfig,
 
 	// Post the swap parameters to the swap server. The response contains
 	// the server revocation key and the swap and prepay invoices.
-	logger.Infof("Initiating swap request at height %v", currentHeight)
+	initiationTime := time.Now()
+	logger.Infof("Initiating swap request at height %v, time %v",
+		currentHeight, initiationTime)
+
+	// We'll take our current time and add our max swap time to it.  This
+	// will be given to the server for it to use as the latest swap
+	// publication time.
+	maxSwapTimestamp := initiationTime.Add(
+		time.Duration(request.MaxSwapWaitTimeSec) * time.Second,
+	)
```

Comment:
I think that it may be better to perform this calculation in the cli or rpc caller, to keep the flexibility to pass in an absolute time.

Reply:
Done!